### ISSUE TITLE
Unable to add Plugins and issue with mission not accepting even after accepting

### ISSUE THREAD

**Issue by FireWolf74** [endless-sky/endless-sky#3760]
I am unable to add plugins of either the steam side or on the Linux side.  Also everytime I go to accept a new mission I click it but the mission never shows in my mission field.  But it is removed from the mission selection.

One last thing the target selection for any enemy ship is way to fast.

**Comment by jafdy** [comment]
Can you provide screenshots as it is quite hard to tell what you are trying to say here.

Targeting of ships is a separate issue, also define "too fast".

**Comment by temtemy** [comment]
Have you followed the instructions in the wiki? Adding plugins is not that hard:

### GNU/Linux ###
Place the plugin folder in either of these:
* /usr/share/endless-sky/plugins/
* ~/.local/share/endless-sky/plugins/ (if you use a gui file manager you may need to show hidden files)

### Windows ###
Place the plugin folder in either of these:
* plugins\ (in the same folder as the Endless Sky executable)
* C:\Users\yourusername\AppData\Roaming\endless-sky\plugins\

### Mac OS X ###
Place the plugin folder in either of these:
* Content/Resources/plugins/ (within the application bundle)
* ~/Library/ApplicationSupport/endless-sky/plugins

(~ means your home folder)

**Comment by temtemy** [comment]
And also, please separate each "bug" you found into different issues.

**Comment by FireWolf74** [comment]
KiLEdEnNis
I use Linux and Steam versions only

Temtemy
Well on the Linux side the info about Linux saving is incorrect.  Mine don't have a plugin folder.  And on the Steam side I can add any DLCs that Steam has not approved.

Jafdy,
Like when you jump into a system or when you leave a planet you get attacked and killed in less than 1 second.

Guys I normally don't ask for help unless I have nailed a brick wall.  I work on PCs and Networks daily and I made my own computer.

**Comment by tehhowch** [comment]
For working on PCs and Networks daily, you've provided a rather lackluster bug report.
1) Multiple issues in the same "ticket", and no evidence of reviewing previous Issues
2) No reproduction steps (or screenshots to convey graphical issues).
3) No details about how you installed your linux version - this game is present on PPAs, as a package, and as a DIY build from source. Depending on the steps you / the pm performed, you may or may not have configured `/usr/share`. I needed to perform an `install` step with SCONS before the application runtime icon and `/usr/share` were configured - and I still use `~/.local/share/endless-sky` anyway.

Note: this is not a discussion forum, this is an Issue tracker, for bug reports and/or fleshed-out feature requests: you (and everyone else) are expected to review existing Issues to see if what you wish to address has arisen previously. Discussion forums are available on Google Groups and Steam, and unofficially on Reddit and via Discord.

**Comment by FireWolf74** [comment]
tehhowch,

I am glad to see that you are trying to tell me how to fix a Debian for an RPM system. (if that was harsh sorry but Linux version are not only Debian).  Normally when someone is talking RPM they normally just say Linux because Debian includes Ubuntu and ever other flavor that is created from that.

Second off until Linux Kernel 3.12 I think the ./local/share but now it is usr/share.  Now within that share folder I have a folder endless-sky and in the folder I have the below Screenshot.  This is only for the one I installed from the software package center
![screenshot from 2018-06-07 20-57-02](https://user-images.githubusercontent.com/33334258/41133596-daf2dcde-6a95-11e8-91f7-8442b2e5a559.png)

**Comment by FireWolf74** [comment]
Now I don't have any idea where the steam one is.

**Comment by FireWolf74** [comment]
Jafdy,
Here you go

https://drive.google.com/file/d/1scFuntBVF0AbzWLZ5p86j1oT1FUdU7jJ/view?usp=sharing

**Comment by MCOfficer** [comment]
i can't see anyhing concerning either of your issues in that video.

**Comment by jafdy** [comment]
I am glad I asked for clarification as your issue is entirely different from what I though it was.
Enemy ships fire at the closest target, if that is you then they fire at you. You video shows a ship taking off from a planet, that is meant to happen. So there appears to be no bug and no issue.

Also you can create the plugin folder.

**Comment by FireWolf74** [comment]
jafdy,
I will agree if that is the game Mechanics ok but if mupility units deploy at the same time what are the Mechanics on that?  
Also I know I can make  Plugins Folder for the Linux version but where is the plugins folder for the steam folder?

MCOfficer,
As I was saying to jafdy I understand with the single attack game mechanic but if you have say 5 SS Carriers and Gunboats why are the Falcons and the enemy Gunboats and Carriers attacking someone with a Nest or even a Freighter leaving or entering a system?
I agree this sounds like I'm winning but if any of the Game Mechanics coding is flawed it might need to be looked at and fix.

**Comment by FireWolf74** [comment]
Sorry but to me a Freighter or a Nest is not a juicy target when surrounded by Carriers.

**Comment by jafdy** [comment]
. . . For someone who "works on PCs daily" you seem to have a very large issue with adding a file. I can't think of anything anyone could say that would make you realise that you can add files yourself and don't need steam to do it for you. 
And I am not going to continue discussing this until you sort this jumbled mess into separate issues with one bug (including screenshots) or issue (with well thought out argument). And you can edit your posts with the little pen icon in the corner, that is 130 people that don't then get extra emails they don't want.

**Comment by KiLEdEnNis** [comment]
> Sorry but to me a Freighter or a Nest is not a juicy target when surrounded by Carriers.

eh? imagine you're captain of the Black Pearl and you see some US Destroyers, Cruisers etc. and a scarcely-armed freighter full of cargo
not considering those technologically advanced ships will pound you in the butt, which ship would you prefer to board and plunder? some heavy armed killing machine or the lightly armed cargo ship

as for the Nest scenario, target selection is: more cargo capacity? better target & more weapons? worse target
while a Nest lacks cargo space, it at least has maybe a third of the armament a warship has (taking into consideration not only the amount of weapons, but also their strengths)

**Comment by FireWolf74** [comment]
Let me see if I can dumb it down further for you jafdy.  Where is the Steam Version (ie. the Steam folder) on the Linux machine.  I have looked but it is no where to be found.  There is that Plain enough for you?

**Comment by KiLEdEnNis** [comment]
such a condescending tone isn't really appreciated here
it's your fault for not being clear enough, not jafdy's for not understanding your unclear and limited details

please refrain from doing so again, as we're all here in our spare time for fun and and to help work on a game we like

**Comment by FireWolf74** [comment]
Wrzlprnft,
So let me get this straight Zero plugins that is within the Github repos they can't be used in the Steam Version at all????

**Comment by jafdy** [comment]
Right click on ES -> properties -> local files -> browse local files -> create "plugin" folder in correct place.
OR
Navigate to you save file and place the plugin folder there.

**Comment by FireWolf74** [comment]
jafdy,

thanks a million man.

**Comment by temtemy** [comment]
@jafdy Placing the plugin folder in the location of the save file (saves) won't work.

**Comment by jafdy** [comment]
One level up, there is a plugin folder there that works. It's the second line in the instruction you posted.

**Comment by temtemy** [comment]
@jafdy Ah, it is the "plugins" folder you are referring to, right? Not the plugin itself?

**Comment by KiLEdEnNis** [comment]
he means the plugins folder, which is in the same folder as saves; both folders are in endless-sky
put any plugin IN this plugin folder and it should load
